src/compiler.rs

```diff
@@ -324,8 +324,14 @@ impl AmberCompiler {
 
     #[cfg(not(windows))]
     fn find_bash() -> Option<Command> {
-        let mut command = Command::new("/usr/bin/env");
-        command.arg("bash");
-        Some(command)
+        if env::var("GITHUB_ACTIONS_BASH_CONTAINER").is_ok() {
```

Comment:
@hdwalters it isn't better that this code there is only on a debug build like you did for another PR?

Reply:
I don't like debug and release builds having different functionality.  I reverted this behaviour in your documentation usage code when I did the revamped CLI; it's now enabled with a new command line option `amber docs --usage`.

Reply:
Although in this case, it may make sense. I suggest asking for opinions on the maintainers group.

Reply:
It will be helpful if there is a CLI option to choose other bash executable or kind of entry point. Some other environment, for example [Termux](https://termux.dev/)(an Android terminal emulator) has no `/usr/bin/env` so always fails to run `amber eval`.

Reply:
Perhaps, but that's out of scope for this PR. AIUI, @Mte90's point was that since your `GITHUB_ACTIONS_BASH_CONTAINER` change was intended to support `cargo test` in a Docker container, it should probably not be enabled for release builds. If we agree on this, we could do something like:
```
#[cfg(not(windows))]
#[cfg(debug)]
fn find_bash() -> Option<Command> {
    // Use docker or /usr/bin/env...
}

#[cfg(not(windows))]
#[cfg(not(debug))]
fn find_bash() -> Option<Command> {
    // Use /usr/bin/env...
}
```

Reply:
Yep that's my idea.